apps/notifications/src/notifications/notification.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { Exclude } from 'class-transformer';
+
+import { BaseEntity, NotificationsEnum } from '@vp-clients-app/common-pkg';
+
+@Entity('notifications', { schema: 'notifications' })
+export class NotificationEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ enum: NotificationsEnum })
+  type: string;
+
+  @Column({ name: 'user_id' })
+  userId: string;
+
+  @Column({ name: 'is_read' })
+  isRead: string;
+
+  @Column()
+  payload: string;
```

Comment:
why string? 

Should be typed 

payload: T

Reply:
interface Appointment {};

class AppointmentEntity implements Appointment {}

class NotificationAppontmentCreated <{ payload: Appointment }>